1:30 - Update Events (#890)
FusionAuth/fusionauth-site: site/docs/v1/tech/events-webhooks/event-bodies/user-password-reset-send.adoc

:type: user.password.reset.send
== User Password Reset Send

[NOTE.since]
====
This event has been available since 1.30.0
====

This event is generated when a reset password request has been sent.

[source,shell]
.Event type
[subs="attributes"]
----
{type}
----

=== Event Scope

include::../_tenant-scoped-event.adoc[]

> 

Reviewer: Add `Transaction Compatible` section.

=== Event Body

[.api]
[field]#event.createInstant# [type]#[Long]#::
The link:/docs/v1/tech/reference/data-types/#instants[instant] that the event was generated.

[field]#event.id# [type]#[UUID]#::
The unique Id of the event. You may receive an event more than once based upon your transaction settings. This Id may be used to identify a duplicate event.

include::_event-info.adoc[]

[field]#event.tenantId# [type]#[UUID]#::
The unique tenant identifier. This value may not be returned if not applicable.

[field]#event.type# [type]#[String]#::
The event type, this value will always be `{type}`.

[field]#event.user# [type]#[Object]#::

> The user that has initiated the reset password send request. See the link:/docs/v1/tech/apis/users/[Users API] for property definitions and example JSON.

Reviewer: ```suggestion
The user for whom the reset password send request was initiated. See the link:/docs/v1/tech/apis/users/[Users API] for property definitions and example JSON.
```
